### ISSUE TITLE
Resculpt not working

### ISSUE THREAD

**Issue by Sareya84** [peterhaneve/ONIMods#50]
When pressing the "Resculpt" button, a Duplicant should be tasked to resculpt the intended sculpture; however, no such event is taking place. Currently, I have to deconstruct the sculpture and get a dupe to make a new one.

**Comment by peterhaneve** [comment]
Can you take a screenshot of the _Errands_ tab of the sculpture to see if anything shows up? There is a base game bug where occasionally some chores will not be performed without saving and reloading, and they mysteriously show a blank tab on that page.

**Comment by Sareya84** [comment]
Sure! I've attached it below. I clicked the "Resculpt" button in and out of the Errands tab. I don't have the option to choose what priority I want the resculpt to be and it doesn't show up in the Errands tab. I tried saving and reloading a few times, but nothing changed. Thank you so much for looking in to this! =)

![ONI Resculpt Errands](https://user-images.githubusercontent.com/65376291/82103069-86f33c00-96c6-11ea-9b18-0a6efcce60dc.jpg)
